docs/USAGE.md

```diff
@@ -0,0 +1,261 @@
+# Using Confluent for VS Code
+
+### Work with your Confluent Cloud resources
+
+<!-- <gif showing clicking sign in to confluent cloud, entering credentials and then getting redirected back to vscode -->
+<!-- will resources populated, then clicking through an org -> env -> cluster (displays topics), then SR (displays schema)> -->
+
+Log in to your Confluent Cloud account from the Confluent for VS Code extension by clicking on "Sign in to Confluent Cloud", and explore your Confluent Cloud resources from right within VS Code.
+
+> [!NOTE]
+> If you would like to connect to your Apache Kafka® cluster on Confluent Cloud and/or Confluent Cloud Schema Registry cluster
+> using [API Key (user account or service account)](https://docs.confluent.io/cloud/current/security/authenticate/workload-identities/service-accounts/api-keys/overview.html#api-keys-and-ccloud-accounts), [Mutual TLS](https://docs.confluent.io/cloud/current/security/authenticate/workload-identities/identity-providers/mtls/overview.html), or [OAuth/OIDC](https://docs.confluent.io/cloud/current/security/authenticate/workload-identities/identity-providers/oauth/overview.html), then head to <anchor tag> for how to do that using the extension.
+
+
+### Bootstrap streaming projects from Confluent-provided templates
+
+<!-- <gif showing clicking button, clicking on a template, fill in values, opening the project, and running it> -->
+
+Confluent for VS Code offers project templates designed to accelerate your development process. These ready-to-use templates are tailored for common development patterns, allowing you to quickly launch new projects with minimal configuration.
+
+> [!NOTE]
+> We will continue to add new templates as well as improving existing ones, however, please [file an issue]() or [start a discussion]() if you find bugs/issues in existing templates or have proposals for new templates that could benefit the broader community.
+
+### Accelerate local development against Kafka and Confluent Schema Registry using Docker
+
+> [!NOTE]
+> Pre-requisites: You must have Docker Engine (version v24.0 or later) installed on your machine. Please follow the instruction [here](https://docs.docker.com/engine/install/) to install Docker Engine.
+
+Using just a single click, you can easily start a Kafka and Confluent Schema Registry container on your
+local machine (via Docker).
+
+1. Click the green play button to begin the Local resources quickpick flow
+1. Choose whether you want to create just a Kafka cluster or Kafka and Schema Registry, both
+1. Enter the number of brokers you want in the Kafka cluster (this determines the maximum replication factor for the Kafka topics)
+1. Hit OK and wait for the resources to become available
+1. Now, you can leverage all the features of the extension to work with the Local resources you just started.
```

Comment:
Kind of feels like we could combine these two like "Once the resources are available, you'll be able to ...." but I'm not sure on any better wording

Reply:
Doesn't look great in dark mode, but definitely better than the long text.

![Screenshot 2025-03-14 at 2 38 15 PM](https://github.com/user-attachments/assets/420452f1-1c0e-4747-8037-9d711ed9d2b1)